operations/node_management/Removing_a_Liquid-cooled_blade_from_a_System.md

```diff
@@ -3,42 +3,46 @@
 This procedure will remove a liquid-cooled blades from an HPE Cray EX system.
 
 ## Perquisites
--   The Cray command line interface \(CLI\) tool is initialized and configured on the system.
 
--   Knowledge of whether DVS is operating over the Node Management Network (NMN) or the High Speed Network (HSN).
+- The Cray command line interface \(CLI\) tool is initialized and configured on the system.
 
--   The Slingshot fabric must be configured with the desired topology for desired state of the blades in the system.
+- Knowledge of whether DVS is operating over the Node Management Network (NMN) or the High Speed Network (HSN).
 
--   The System Layout Service (SLS) must have the desired HSN configuration.
+- The Slingshot fabric must be configured with the desired topology for desired state of the blades in the system.
 
--   Check the status of the high-speed network (HSN) and record link status before the procedure.
+- The System Layout Service (SLS) must have the desired HSN configuration.
 
--   The blades must have the coolant drained and filled during the swap to minimize cross-contamination of cooling systems.
-    - Review procedures in *HPE Cray EX Coolant Service Procedures H-6199*
-    - Review the *HPE Cray EX Hand Pump User Guide H-6200*
+- Check the status of the high-speed network (HSN) and record link status before the procedure.
+
+- The blades must have the coolant drained and filled during the swap to minimize cross-contamination of cooling systems.
+  - Review procedures in *HPE Cray EX Coolant Service Procedures H-6199*
+  - Review the *HPE Cray EX Hand Pump User Guide H-6200*
 
 ## Procedure
 
```

Comment:
The new numbering here might be a little confusing for readers. It might be useful to add an indicator in the level 3 headings to show that this procedure should be followed in order.

Reply:
I applied all of your suggestions, with adding the `Step` prefix to the level 3 headings. Anything else you think I should so?